app/models/recipe.rb

```diff
@@ -2,4 +2,10 @@ class Recipe < ApplicationRecord
   belongs_to :user
   has_many :recipe_foods
   has_many :foods, through: :recipe_foods
+
+  validates :name, presence: true
```

Comment:
### Optional
Validation is not apart of project specifications. It is good that you added one. It would be good to add more specially for times to make sure they are number and positive number.